packages/x-components/src/views/Layout.vue

```diff
@@ -1,5 +1,6 @@
 <template>
   <div>
+    <Tagging :consent="false" />
```

Comment:
The session id is created when doing a tracking request, at this moment there are no tracking request so you are not going to see it.

Reply:
I know, I know and because of that we could even remove this component from the PR. What I meant was to leave it as true in order to see later in the requests, but whatever you want.

Reply:
I will leave it as it is and we can change it in the future if we need it